communication/README.md

```diff
@@ -44,13 +44,13 @@ Note that commercial content is allowed, unsolicited commercial content mostly i
 
 Examples of inappropriate content:
 - Posting unsolicited content of a commercial nature on Slack or other community forums
-- Messaging people individually on a platform with content of an unsolicited commercial nature
+- Messaging people individually on a platform with the content of an unsolicited commercial nature
 - Unsolicited pitching of commercial products during a Kubernetes meeting
 
 Examples of appropriate content:
 - Asking about commercial products in an appropriate channel. For example most clouds have a channel in Slack, asking how to use GKE on the GKE channel or AKS on the Azure channels is fine.
 - "Does anyone have experience with project foo?" is fine
-- Some OSS projects are also hosted on the Kubernetes Slack that also have a commercial offering, these are allowed.
+- Some OSS projects are also hosted on the Kubernetes Slack which also has a commercial offering, these are allowed.
```

Comment:
again, original is correct; replacement is not.

In general, in English, "that" is preferred to "which" in most usages.